src/GovUk.Education.ExploreEducationStatistics.Publisher/Services/FileStorageService.cs

```diff
@@ -40,34 +46,53 @@ public FileStorageService(IConfiguration configuration, ILogger<FileStorageServi
         public async Task CopyReleaseFilesToPublicContainer(CopyReleaseFilesCommand copyReleaseFilesCommand)
         {
             var privateContainer =
-                await FileStorageUtils.GetCloudBlobContainerAsync(_privateStorageConnectionString,
+                await GetCloudBlobContainerAsync(_privateStorageConnectionString,
                     PrivateFilesContainerName);
             var publicContainer =
-                await FileStorageUtils.GetCloudBlobContainerAsync(_publicStorageConnectionString,
+                await GetCloudBlobContainerAsync(_publicStorageConnectionString,
                     PublicFilesContainerName);
 
-            var sourceDirectoryPath = AdminReleaseDirectoryPath(copyReleaseFilesCommand.ReleaseId);
-            var destinationDirectoryPath =
-                PublicReleaseDirectoryPath(copyReleaseFilesCommand.PublicationSlug, copyReleaseFilesCommand.ReleaseSlug);
-
-            if (copyReleaseFilesCommand.ReleaseSlug != copyReleaseFilesCommand.PreviousReleaseSlug)
+            // Slug may have changed for the amendment so also remove the previous contents if it has
```

Comment:
Are we allowed to change the slug?  I thought that was important to stay the same from the public front end perspective?